src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import menuicon from '../../../assets/nav/menuIcon.png';
+import closeicon from '../../../assets/nav/closeIcon.png';

```

Comment:
can we go with camelCase `menuIcon` and `closeIcon`.
```suggestion
import menuIcon from '../../../assets/nav/menuIcon.png';
import closeIcon from '../../../assets/nav/closeIcon.png';

```

Reply:
change https://github.com/Sakshi-Simform/portfolio/pull/2/commits/7e0533de61a384abeda8711fdb99c5587db59551